test/test.js

```diff
@@ -25,12 +25,13 @@ describe('ipfs node api', function () {
     ipfsd.disposable(function (err, node) {
       if (err) throw err
       log('ipfs init done')
+      ipfsNode = node
 
-      node.startDaemon(function (err, ignore) {
+      ipfsNode.startDaemon(function (err, ignore) {
         if (err) throw err
         log('ipfs daemon running')
 
-        ipfs = ipfsApi(node.apiAddr)
+        ipfs = ipfsApi(ipfsNode.apiAddr)
```

Comment:
are these changes necessary? or just for readability? 

maybe i'm not seeing the js-scoping issue at hand, but afaict it is fine to do as it was?